CMakeLists.txt

```diff
@@ -197,6 +197,7 @@ if(BUILD_TESTING)
                      EXCLUDE_FROM_ALL)
   endif()
 
+  enable_language(CXX)
```

Comment:
you should try to move it just after `if(BUILD_TESTING)` line 162 so googletest is build with support enabled